dynamiq/storages/vector/pinecone/pinecone.py

```diff
@@ -50,6 +50,7 @@ def __init__(
         connection: Pinecone | None = None,
         client: Optional["PineconeClient"] = None,
         index_name: str = "default",
+        content_key: str = "content",
```

Comment:
I don't see here `embedding_key`
`embedding = pinecone_doc["values"]` - `values` here, it seems `embedding_key`